includes/Core/Util/Migration_1_119_0.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Migration for 1.119.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use Google\Site_Kit\Modules\Analytics_4\Settings as Analytics_Settings;
+
+/**
+ * Class Migration_1_119_0
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Migration_1_119_0 {
+	/**
+	 * Target DB version.
+	 */
+	const DB_VERSION = '1.119.0';
```

Comment:
According to the ACs, let's aim to only include this in the same release where #7932 goes out. From the current progress, it looks like it could be 1.121.0. We may need to update this later on again if #7932 gets pushed back further.

Let's update this to 1.121.0 for now including the class name, comments, and tests.